Tests/XCUITests/BookmarkingTests.swift

```diff
@@ -123,7 +124,7 @@ class BookmarkingTests: BaseTestCase {
         bookmark()
 
         // There should be a bookmark
-        navigator.goto(MobileBookmarks)
```

Comment:
It looks like `MobileBookmarks` is not applicable. Has the UI been changed?

Reply:
Best way to know is to manually follow the user workflow manually on the simulator. I think the UI changes, right, and now all bookmarks are in the general panel except for the Desktop bookmarks and unless you create a new folder.

Reply:
<img src="https://user-images.githubusercontent.com/1740517/199541873-8165cccc-7ea1-4d59-b81c-1af4ab45bc18.png" width="300" alt="Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-11-02 at 12 10 04"/>

This is the `LibraryPanel_Bookmarks` screen. There supposed to be a "Mobile Bookmarks" link to the `MobileBookmarks` screen according to the screen graph.

https://github.com/mozilla-mobile/firefox-ios/blob/main/Tests/XCUITests/FxScreenGraph.swift#L454

It'd be great if you can If you can show me the `MobileBookmarks` screen.

Reply:
I think that's the right screen, the mobileBookmarks folder was removed. Sorry I did not explain that clearly in my previous comment

Reply:
💡I see. Is the code for `MobileBookmarks` from the `FxScreenGraph` would be unused then. We can remove such unused code in a separate PR. 🙂

Reply:
Yes I'm almost sure that it can be removed. You can do it here or in a follow up FxScreenGraph PR :)

Reply:
Just to add a bit of context, the `MobileBookmarks` is now what you see as the root folder (whereas before we needed to click on a "Mobile Bookmarks" folder to see the mobile bookmarks). In the design [here](https://www.figma.com/file/SO8Cd5lnmgNqJQY9KpORc9/History%2C-Bookmarks%2C-Downloads?node-id=2117%3A58734) you can see what it looked like before.

Reply:
Thanks for confirming that @lmarceau and for sharing the design, I could not find it!